### DISCUSSION TITLE
Azure Powershell Subscription no longer displaying in the prompt

### DISCUSSION THREAD

**Discussion by colinrippeyfinarne** [JanDeDobbeleer/oh-my-posh#3196] [Troubleshoot] [answered]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I've installed the latest version onto a new laptop and when I copied over my custom theme the segment that presented the current Azure Powershell subscription is no longer displayed.

The relevant segment snippets are:

`  "blocks": [
    {
      "alignment": "left",
      "newline": true,
      "segments": [
        {
          "type": "az",
          "style": "powerline",
          "foreground": "p:yellow",
          "template": "{{ .User.Name }} "
        },
        {
          "type": "az",
          "style": "powerline",
          "foreground": "p:white",
          "template": ">p {{ .Name }} ",
          "properties": {
            "source": "pwsh"
          }
        },
        {
          "type": "az",
          "style": "powerline",
          "foreground": "p:green",
          "template": ">a {{ .Name }} "
        },
        {
          "foreground": "p:blue",
          "style": "plain",
          "properties": {
            "style": "full"
          },
          "template": "{{ .Path }} ",
          "type": "path"
        },
        {
          "properties": {
            "branch_icon": "",
            "fetch_status": true,
            "fetch_stash_count": true,
            "fetch_worktree_count": true,
            "fetch_upstream_icon": true
          },
          "style": "plain",
          "template": "<p:yellow>git</>:({{ if or (.Working.Changed) (.Staging.Changed) (gt .StashCount 0) }}<p:magenta>{{ .HEAD }}</>{{ else }}<p:green>{{ .HEAD }}</>{{ end }}{{ if or (gt .Ahead 0) (gt .Behind 0)}}<p:cyan> {{ .BranchStatus }}</>{{ end }}{{ if .Staging.Changed }}<p:green> {{ .Staging.String }}</>{{ end }}{{ if .Working.Changed }} <p:red>{{ .Working.String }}</>{{ end }})",
          "type": "git"
        }
      ],
      "type": "prompt"
    },
`

I see the current az cli active subscription but no longer see the pwsh one, has anything changed recently or do I need to update my segments?

### Theme

Custom

### What OS are you seeing the problem on?

Windows, Linux

### Log output

```Shell
ConsoleTitle(false) -   0 ms - 
az(true)    -   0 ms - [38;2;229;192;123m<redacted> [0m
az(false)   -   0 ms - 
az(true)    -   0 ms - [38;2;152;195;121m>a <redacted> [0m
path(true)  -   1 ms - [38;2;97;175;239mC:\git\test [0m
git(true)   -  26 ms - [38;2;229;192;123mgit[0m[37m:([0m[38;2;198;120;221mmain[0m[37m [0m[38;2;224;108;117m?3[0m[37m)[0m
exit(true)  -   1 ms - [38;2;152;195;121m(0) [0m[38;2;220;223;228m> [0m
```

**Comment by JanDeDobbeleer** [comment, marked as answer]
@colinrippeyfinarne yes, you need to make sure `$env:POSH_AZURE_ENABLED = $true` is added to your profile after initialising oh-my-posh.

**Comment by colinrippeyfinarne** [comment]
Thank you again, I had that line in my profile but I had it *before* the line that called oh-m-posh init. Working again now.
